mathics/builtin/procedural.py

```diff
@@ -443,6 +450,26 @@ def eval_with_false_and_other(self, condition, t, f, u, evaluation):
         else:
             return u.evaluate(evaluation)
 
+    def to_sympy(self, expr, **kwargs):
+        if len(expr.elements) == 3:
+            sympy_cond = expr.elements[0].to_sympy(**kwargs)
+            sympy_true = expr.elements[1].to_sympy(**kwargs)
```

Comment:
These conversions are not necessary if cond is not a valid condition. I'm would put this and the next line inside the conditional containing the successful  return.

Reply:
Apart from this detail, LGTM

Reply:
@bdlucas1 This is my only comment on this. In any case, we can merge this and fix the order later.